test/prompt/invocation_layer/test_hugging_face.py

```diff
@@ -522,6 +522,40 @@ def test_generation_kwargs_from_invoke():
     mock_call.assert_called_with(the_question, {}, {"do_sample": True, "top_p": 0.9, "max_length": 100}, {})
 
 
+@pytest.mark.unit
+def test_max_length_from_invoke(mock_auto_tokenizer, mock_pipeline, mock_get_task):
+    """
+    Test that max_length passed to invoke are passed to the underlying HF model
+    """
+    the_question = "What does 42 mean?"
+    # test that generation_kwargs are passed to the underlying HF model
+    layer = HFLocalInvocationLayer()
+    layer.invoke(prompt=the_question, generation_kwargs={"max_length": 200})
```

Comment:
Let's use `query` instead of `the_question`:
```suggestion
    query = "What does 42 mean?"
    # test that generation_kwargs are passed to the underlying HF model
    layer = HFLocalInvocationLayer()
    layer.invoke(prompt=query, generation_kwargs={"max_length": 200})
```